### ISSUE TITLE
Famicom Disk System & Satellaview

### ISSUE THREAD

**Issue by sysconf16** [webrcade/webrcade#44]
As the option to play NES & cartridge-based Famicom games is already available, opening access to FDS exclusives and revisions (e.g. Super Mario Bros. 2 and Metroid/The Legend of Zelda with FM sound) would be fantastic.

In addition, the majority of the Satellaview catalogue does not work despite snes9x support for the add-on.
All games either stutter, hang or display garbled graphics.

**Comment by raz0red** [comment]
Yes, something I plan to add and should be fairly straight-forward. 
Thanks for the suggestion!

**Comment by raz0red** [comment]
FDS is part of 0.2.0
